src/lib/server/shop/addToCart/reservations.ts

```diff
@@ -229,6 +229,7 @@ const moveReservationsToCart = async (
   reservationsToMove: ConsumableReservation[],
   updateOrder = true,
 ) => {
+  if (reservationsToMove.length === 0) return;
```

Comment:
Do you ever encounter this case in the wild? Or should it error in some kind of way so that we know that it happened?

Reply:
Yes this is quite common to be true. If `moveQueueToCart` or `moveQueueForwardOneStep` is called when there is no active queue this if statement will be true. So we stop it here really just for performance reasons, avoiding unecessary calls to the database which doesn't do anything.